mongodbatlas/resource_mongodbatlas_cluster.go

```
@@ -1010,6 +1010,10 @@ func resourceMongoDBAtlasClusterImportState(ctx context.Context, d *schema.Resou
 		log.Printf(errorClusterSetting, "name", u.ID, err)
 	}
 
+	if err := d.Set("cloud_backup", u.ProviderBackupEnabled); err != nil {
+		return nil, fmt.Errorf("couldn't import cluster %s in project %s, error: %s", *name, *projectID, err)
```

do we want to throw the same error as `Clusters.Get`? should probably use different wording here.

---

Added additional content relating error to backup configuration